src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -1369,16 +1287,146 @@ else if (isPostGISDataStore(dataStore)) {
                 if (error != null) {
                     error = e;
                 }
-                LOGGER.log(Level.WARNING, "Error closing transaction",e);
+                LOGGER.log(Level.WARNING, "Error closing transaction", e);
             }
         }
-        
         // finally, throw any error
         if (error != null) {
             throw error;
         }
     }
 
+    private Exception copyFeaturesHighLevelAPI(ImportData data, ImportTask task,
+            DataStoreFormat format, DataStore dataStore, Transaction transaction,
+            String featureTypeName, String uniquifiedFeatureTypeName,
+            FeatureDataConverter featureDataConverter, VectorTransformChain tx) throws IOException {
```

Comment:
I updated the name.

I don't think untangling is going to help at all - there's just going to be another method added that pretty much does what this one does.